packages/eslint-plugin/lib/config/jest.ts

```diff
@@ -1,31 +1,20 @@
-import { reactTestFiles, testFiles } from "../utils/patterns";
-
 import type { Linter } from "eslint";
+import { concat } from "eslint-flat-config-utils";
+import jestPlugin from "eslint-plugin-jest";
+import { reactTestFiles, testFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: [...testFiles, ...reactTestFiles],
-            plugins: ["jest"],
-            env: {
-                es2024: true,
-                node: true,
-                browser: true,
-                commonjs: true,
-                jest: true
-            },
-            extends: ["plugin:jest/recommended"],
-            rules: {
-                // Prefer spies to allow for automatic restoration
-                "jest/prefer-spy-on": "error",
-                // Gives better failure messages for array checks
-                "jest/prefer-to-contain": "error"
-            }
+const config: Linter.FlatConfig[] = await concat(
+    {
+        name: "Workleap/Jest",
+        files: [...testFiles, ...reactTestFiles],
+        ...jestPlugin.configs["flat/recommended"],
```

Comment:
Could something similar to `import packageJsonPluginRecommended from "eslint-plugin-package-json/configs/recommended";` also work for this one?

Reply:
No, "flat/recommended" is actually an object key, and not a file path, so it doesn't know how to import that. The package.json one is a true file path, and it doesn't actually have a default export at the root. There's no consistency between these libraries at the moment, even the one ones that have updated.